src/gam.py

```diff
@@ -1994,7 +1997,7 @@ def getCourseAttribute(myarg, value, body, croom, function):
   elif myarg in ['owner', 'ownerid', 'teacher']:
     body['ownerId'] = normalizeEmailAddressOrUID(value)
   elif myarg in ['state', 'status']:
-    validStates = _getEnumValuesMinusUnspecified(croom._rootDesc['schemas']['Course']['properties']['courseState']['enum'])
+    validStates = _getValidCourseStates(croom)
```

Comment:
I don' really see the point in calling 2 one line functions here, just call _getEnumValuesMinusUnspecified...